english/chopper/get-to-it.md

```diff
@@ -0,0 +1,3 @@
+The bananas were hot while the tea was cold. What craziness is this? Give me a current bun and be done with it all, I thought. At least I don't have to get to the [chopper](http://s2.quickmeme.com/img/98/982c902d16cad216960055a63f97c0f776ef6745d52c9713a2c5e8c5a07ce115.jpg)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.